### PR TITLE
Remove mentions of Typed - replaces @@@note of classic/typed with an includes

### PR DESCRIPTION
**Pull request by helena**
## References
https://github.com/akka/akka/issues/24717

## Changes
Replaces all of these scattered in most of the /paradox root markup files with an includes

```
@@@ note

Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs, 
known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use 
the classic APIs. It is also possible to use Akka Typed together with classic actors within the same 
ActorSystem, see @ref[coexistence](typed/coexisting.md). For new projects we recommend using the new Actor APIs.

@@@
```

### CODE REVIEW

File: akka-docs/src/main/paradox/mailboxes.md [hunk 1]
```diff
@@ -1,16 +1,8 @@
...
 
-@@@ note
-
-Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs, 
-known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use 
-the classic APIs. It is also possible to use Akka Typed together with classic actors within the same 
-ActorSystem, see @ref[coexistence](typed/coexisting.md). For new projects we recommend using the new Actor APIs.
-
+@@include[includes.md](includes.md) { #actor-api }
 For the new API see (FIXME https://github.com/akka/akka/issues/27124).
```
Comment (patriknw) [inline akka-docs/src/main/paradox/mailboxes.md]: I think that link can be added now typed/mailboxes.md

**Review by patriknw — APPROVED** [review]
LGTM, possibly one link can be added...

File: akka-docs/src/main/paradox/mailboxes.md (hunk 1 above)
Comment (helena) [reply akka-docs/src/main/paradox/mailboxes.md]: added

**Review by raboof — APPROVED** [review]

File: akka-docs/src/main/paradox/index-actors.md [hunk 2]
```diff
@@ -1,15 +1,6 @@
...
-
-Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs, 
-known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use 
-the classic APIs. It is also possible to use Akka Typed together with classic actors within the same 
-ActorSystem, see @ref[coexistence](typed/coexisting.md). For new projects we recommend using the new Actor APIs.
-
-For the new API see @ref[Actors](typed/actors.md).
-
-@@@
+@@include[includes.md](includes.md) { #actor-api }
```
[the remark this answers is not in the document]
Comment (helena) [reply akka-docs/src/main/paradox/index-actors.md]: That is in the included snippet already.

File: build.sbt [hunk 3]
```diff
@@ -239,7 +239,12 @@ lazy val docs = akkaModule("akka-docs")
       // Page that recommends Alpakka:
       "camel.html",
       // TODO seems like an orphan?
-      "fault-tolerance-sample.html"
+      "fault-tolerance-sample.html",
+      // FIXME https://github.com/lightbend/paradox/issues/350
+      // for subdirs you can add an /includes/page.html to add snippets
+      // but from a root page.md it can't resolve a typed/page.html from /subdir/page
+      // other include options are however possible from an /includes
```
[the remark this answers is not in the document]
Comment (helena) [reply build.sbt]: I did not write that well, but basically you can't have a root page.md reference a includes/page.md for a snippet needing ../ for its `@ref` to render properly where for the root includes that path is not valid.

File: akka-docs/src/main/paradox/dispatchers.md [hunk 4]
```diff
@@ -1,15 +1,7 @@
...
-
-Akka Classic is the original Actor APIs, which have been improved by more type safe and guided Actor APIs, 
-known as Akka Typed. Akka Classic is still fully supported and existing applications can continue to use 
-the classic APIs. It is also possible to use Akka Typed together with classic actors within the same 
-ActorSystem, see @ref[coexistence](typed/coexisting.md). For new projects we recommend using the new Actor APIs.
-
-For the new API see @ref[Dispatchers](typed/dispatchers.md).
-
-@@@
+@@include[includes.md](includes.md) { #actor-api }s
```
Comment (raboof) [inline akka-docs/src/main/paradox/dispatchers.md]:
```suggestion
@@include[includes.md](includes.md) { #actor-api }
```

File: akka-docs/src/main/paradox/dispatchers.md [hunk 5]
```diff
@@ -1,15 +1,7 @@
...
+For the new API see @ref:[Dispatchers](typed/dispatchers.md).
```
Comment (raboof) [inline akka-docs/src/main/paradox/dispatchers.md]:
```suggestion
```
This was actually already linked on the next line

**Review by raboof — APPROVED** [review]

**Pull request merged**